org.expresslrs.ExpressLRSConfigurator.yml

```diff
@@ -0,0 +1,70 @@
+app-id: org.expresslrs.ExpressLRSConfigurator
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "22.08"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node16
+command: expresslrs-configurator
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
+  - --socket=system-bus
+  - --filesystem=/tmp
+  - --filesystem=/run/udev:ro
+  - --persist=.platformio
+modules:
+  - name: git
+    make-args:
+      - INSTALL_SYMLINKS=1
+      - NO_PERL=YesPlease
+      - NO_TCLTK=YesPlease
+    make-install-args:
+      - INSTALL_SYMLINKS=1
+      - NO_PERL=YesPlease
+      - NO_TCLTK=YesPlease
+    post-install:
+      - find . -type f -name perllocal.pod -delete
+    sources:
+      - type: archive
+        sha256: 814641d7f61659cfbc17825d0462499ca1403e39ff53d76a8512050e6483e87a
+        url: https://mirrors.edge.kernel.org/pub/software/scm/git/git-2.37.3.tar.xz
+        x-checker-data:
+          type: html
+          url: https://mirrors.edge.kernel.org/pub/software/scm/git/
+          version-pattern: git-([\d\.-]+).tar.xz
+          url-template: https://mirrors.edge.kernel.org/pub/software/scm/git/git-$version.tar.xz
+  - name: expresslrs-configurator
+    buildsystem: simple
+    build-commands:
+      - ar x expresslrs-configurator_*.deb
+      - rm expresslrs-configurator_*.deb
+      - tar xf data.tar.xz
+      - cp -r 'opt/ExpressLRS Configurator' '/app/ExpressLRS-Configurator'
+      - chmod -R a-s,go+rX,go-w "/app/ExpressLRS-Configurator"
+      - install expresslrs-configurator.sh /app/bin/expresslrs-configurator
+      - install -Dm644 /app/ExpressLRS-Configurator/resources/assets/icons/512x512.png /app/share/icons/hicolor/512x512/apps/org.expresslrs.ExpressLRSConfigurator.png
```

Comment:
They are actually in the "right place" (for a /usr prefix) in the deb file. Didn't you look at the content?

*hint* `cp -R usr/share /app/share` should do it. This will also copy the .desktop and then for both using `rename-icon` and `rename-desktop-file`

Reply:
also above you just copied a symlink that will not resolve.

Reply:
I only checked the build dir and as I only copied opt, it had no .desktop
I've only started to extract that deb on my own earlier. Still, your instructions doesn't seem correct/enough. It doesn't seem to find the .desktop file.

Reply:
did you ignore the `rename-desktop-file` I mentionned? Or just put it in the worng place and ignored the flaptak-builder warnings?

Reply:
I'm doing 
```
rename-desktop-file: expresslrs-configurator.desktop
rename-icon: expresslrs-configurator
```
and your copy command, which I think should match. I don't think I get a warning.

```
Renaming org.expresslrs.ExpressLRSConfigurator.metainfo.xml to share/appdata/org.expresslrs.ExpressLRSConfigurator.appdata.xml
Renaming expresslrs-configurator.desktop to org.expresslrs.ExpressLRSConfigurator.desktop
Error: Error moving file /home/razze/development/flathub/flathub/.flatpak-builder/rofiles/rofiles-c5Ou1Y/files/share/applications/expresslrs-configurator.desktop: No such file or directory
```

Reply:
Just realized what you meant with "wrong place" I curretly have it at the top/root of the document.

Reply:
that's the right place.

Maybe the `cp` copied in the wrong place. like `/app/share/share` (ie I got it wrong)

Reply:
Yep, that was it. Now I'm getting:

```
WARNING: Binary not found for Exec line in /home/razze/development/flathub/flathub/build-dir/export/share/applications/org.expresslrs.ExpressLRSConfigurator.desktop: "/opt/ExpressLRS Configurator/expresslrs-configurator" %U
error: /home/razze/development/flathub/flathub/build-dir/export/share/icons/hicolor/1024x1024/apps/org.expresslrs.ExpressLRSConfigurator.png is not a valid icon: Image too large (1024x1024). Max. size 512x512
```

Reply:
you might need to use `desktop-file-edit` to set the binary properly.

and remove the 1024x1024 icon (directory)

Reply:
Pushed, thank you